cli/user/README.md

```diff
@@ -94,6 +95,12 @@ All available commands:
 	stream read scoped-stream-name [group-similar] [timeout-in-seconds]: Reads all Events from a Stream and then tails the Stream.
 ```
 
+You can try out commands such as:
```

Comment:
Here I miss again the new section for Kubernetes and explaining more in detail how to configure the tool, which is the main point of the issue associated to this PR.